cmd/create/oidcprovider/cmd.go

```diff
@@ -66,16 +77,18 @@ func run(cmd *cobra.Command, argv []string) {
 
 	// Allow the command to be called programmatically
 	skipInteractive := false
-	if len(argv) == 2 && !cmd.Flag("cluster").Changed {
+	if len(argv) == 3 && !cmd.Flag("cluster").Changed {
```

Comment:
I think we want to leave this as 2 and have another check to disable interactive mode if there are 3 arguments otherwise this will break the existing flow?

Reply:
I fixed the current flow so both would send 3 arguments, this is for when it is called progmmatically from the `create cluster` command